src/models.ts

```diff
@@ -25,6 +25,7 @@ export interface Term {
 }
 
 export interface Constructor {
+    fileName: string;
```

Comment:
?

Reply:
We can probably remove this additional state with how the calls to getPeirceDB() now work.

Reply:
this shouldn't be here